WordPress/gutenberg
# Block Editor: Remove hint in the Settings tab

Fixes #63506
Related to #47670
Similar to #63514

## What?

This PR removes the hint about the Style tab from the Settings tab in the block sidebar.

![image](https://github.com/user-attachments/assets/175b1186-0c55-40aa-a9a5-402fb9f11448)

## Why?

This hint was added to inform users of the change about `InspectorControls` and was shipped in WordPress 6.2. #47670 doesn't say whether this hint will be removed in the future, but since it's been over a year since WordPress 6.2 was released, I think it's safe to remove it now.

## How?

Just remove. The SettingsTabHint component is not exposed externally, so it is safe to delete it directly.

## Testing Instructions

- Run this snippet in the browser console to bring the dismissed pointer back again:
  ```
  wp.data.dispatch( 'core/preferences' ).set( 'core', 'isInspectorControlsTabsHintVisible', true )
  ``
- Insert a block that has Settings and Styles tabs, e.g. the Group block.
- Open the block sidebar and there should be no hints under the Advanced panel.
